mesonbuild/build.py

```diff
@@ -2562,7 +2562,7 @@ def get_all_link_deps(self):
 
     def is_internal(self) -> bool:
         '''
-        Returns True iif this is a not installed static library.
+        Returns True if this is a not installed static library.
```

Comment:
This is not a spelling error: "iff" is short for "if and only if".

Reply:
two I's, not two f's.

They are both wrong, we need a fix for the fix...

Reply:
I'm aware of "iff" but indeed, it's not that.  Since this didn't seem to be used much in the code I went with the pedestrian "if".

Reply:
I'm fine either way, then.

Reply:
There are also several other places with exactly the "[return] True if [...]" pattern so I'll leave it for consistency unless anyone especially feels these ones should be `iff`, or they all should be, or whatever.